x/mongo/driver/topology/topology_test.go

```diff
@@ -548,7 +577,6 @@ func TestTopology_String_Race(t *testing.T) {
 	}()
 
 	go func() {
-		topo.String()
```

Comment:
Why was this removed? This test asserts that the `topology.Topology.String()` function is not vulnerable to data races. To do that, this test launches two goroutines. One modifies `topo.servers` and the other concurrently calls `topo.String()`, which internally accesses `topo.servers`. Removing this call makes removes the core functionality of the test.

Reply:
If your compiler is complaining about it, can you try changing it to `_ = topo.String()`?